interpreter/binary/decode.ml

```diff
@@ -126,6 +126,12 @@ let sized f s =
   require (pos s = start + size) s start "section size mismatch";
   x
 
+let bits i j flags =
```

Comment:
These functions here have the same name as function in encode.ml which do something different. To create a contrast, could you call the two function here `extract_bits from to flags` and `extract_bit pos`? Then the intention of the function is then obvious, and clearly different to the ones in encode.ml